packages/carbon/src/abstracts/BaseModalComponent.ts

```diff
@@ -5,10 +5,22 @@ import type {
 } from "discord-api-types/v10"
 import { BaseComponent } from "./BaseComponent.js"
 
+export type APIFileUploadComponent = {
+	type: 18
```

Comment:
According to the now closed pull request, component type is `19` (`18` is label)